cmd/commands/util.go

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import "strings"
+
+// Implements pflag.Value
+// Custom type so slice values can be represented as a space-separated list of strings in the flag value
+//
+// Example:
+//
+//     var vals []string
+//     defaultVals := []string{"default1", "default2"}
+//
+//     Flags().VarP(newStringSliceValue(strings.Join(defaultVals, " "), &vals), "flagname", "f", "Flag description")
```

Comment:
Why space separated and not comma separated?

Reply:
I arbitrarily picked space-separated. Is comma separated the common thing to do?

Reply:
I'd guess that it's more common! I've never really seen space separated flags tbh!

Reply:
Ok I'll switch it to comma-separated!